jobs/build/signed-compose/rpmdiff.py

```diff
@@ -33,18 +33,31 @@ def rpmdiffs_resolved(advisory):
     rpmdiffs = advisory.externalTests(test_type='rpmdiff')
     completed_diffs = []
     incomplete_diffs = []
+    failed_diffs = []
 
     print("Current RPM Diff status data:")
     pp(rpmdiffs)
 
     for diff in rpmdiffs:
-        if diff['attributes']['status'] in ['INFO', 'WAIVED', 'FAILED', 'NEEDS_INSPECTION']:
+        if diff['attributes']['status'] in ['INFO', 'WAIVED', 'NEEDS_INSPECTION']:
```

Comment:
IIRC, the rpmdiff results from Errata Tool has less number of statuses than what you see on rpmdiff.
For example, `FAILED` in Errata Tool may means `FAILED`, `NEEDS_INSPECTION`, or the rpmdiff test is not finished. For raw rpmdiff status, you need to check `diff['attributes']['external_status']` instead.
I don't have the full list of valid statuses. Be carefully if some statuses are missing.

Reply:
Also, you might want to use `set` rather than `list` in the `in` clause:
``` python
if diff['attributes']['status'] in { 'state1', 'state2' }:
```

Reply:
Well, I got all valid RPMDiff statuses from Errata's source code. I made lists of them for Elliott in `elliott rpmdiff show` PR: https://github.com/openshift/elliott/pull/86/files#diff-fb7e98bd08dd54fd472c838e06d6224cR98-R103

IMO you need add the `PASSED` status here (`INELIGIBLE` is not used in RPMDiff actually).